README.md

```diff
@@ -103,10 +105,15 @@ Of course, now we have to install the extension's dependencies.
 
 One said dependency is `plypythonu`, but luckily that was taken care of when we specified `--with-python` in the PostgreSQL installation. Let's talk instead of `postgis`.
 
-Installing `postgis` is very simple. You also need to create some postigs templates in the db for CARTO to work with:
+Installing `postgis` is very simple. We'll install from source to be sure sure that it will use PostgreSQL 9.5. You also need to create some postigs templates in the db for CARTO to work with:
 
 ```
-brew install postgis
+brew install automake libtool
+cd ~/Documents/workspace
+git clone https://github.com/postgis/postgis.git
+git checkout 2.2.1
+./autogen.sh
+make && make install
```

Comment:
I like to separate these as you may encounter issues with `make` and wouldn't like to run `make install` if so.